contracts/controller.sol

```diff
@@ -114,13 +114,13 @@ contract Controller is IController, Ownable, Transferrable {
     /// @notice is an address an Admin?
     /// @return true if the provided account is an admin.
     function isAdmin(address _account) public view notStopped returns (bool) {
-        return _isAdmin[_account];
+        return _isAdmin[_account] || _isOwner(_account);
```

Comment:
Am not sure about this one. Even though it makes sense, i don't want it to just be slipped through. My guess is that it being more rigid, allows for more controll and better audittings of what the various keys in our hierarchy are signing. 

Thoughts ?

Including you @i-stam, what do you think ?